platform/ios/godot_view.mm

```diff
@@ -205,11 +206,14 @@ - (void)startRendering {
 	if (self.useCADisplayLink) {
 		self.displayLink = [CADisplayLink displayLinkWithTarget:self selector:@selector(drawView)];
 
-		// Approximate frame rate
-		// assumes device refreshes at 60 fps
-		int displayFPS = (NSInteger)(1.0 / self.renderingInterval);
-
-		self.displayLink.preferredFramesPerSecond = displayFPS;
+		if (GLOBAL_GET("display/window/ios/allow_high_refresh_rate")) {
+			self.displayLink.preferredFramesPerSecond = 120;
+		} else {
+			// Approximate frame rate
+			// assumes device refreshes at 60 fps
+			int displayFPS = (NSInteger)(1.0 / self.renderingInterval);
+			self.displayLink.preferredFramesPerSecond = displayFPS;
```

Comment:
This probably can be changed to:
```
self.displayLink.preferredFramesPerSecond = 60;
```

And a branch below changed to:
```
self.animationTimer = [NSTimer scheduledTimerWithTimeInterval:(1.0 / 60) target:self selector:@selector(drawView) userInfo:nil repeats:YES];
```

`renderingInterval` is always set to `1.0 / 60` and is never changed, so probably should be removed at all.